go-chaos/cmd/cluster.go

```diff
@@ -128,32 +145,44 @@ func scaleDownBrokers(k8Client internal.K8Client, port int, brokers int, replica
 	return changeResponse, nil
 }
 
-func requestBrokerScaling(port int, brokers int, replicationFactor int32) (*ChangeResponse, error) {
+func requestBrokerScaling(port int, brokers int, partitionCount int32, replicationFactor int32) (*ChangeResponse, error) {
 	brokerIds := make([]int32, brokers)
 	for i := 0; i < brokers; i++ {
 		brokerIds[i] = int32(i)
 	}
-	return sendScaleRequest(port, brokerIds, false, replicationFactor)
+	return sendScaleRequest(port, brokerIds, partitionCount, false, replicationFactor)
 }
 
-func sendScaleRequest(port int, brokerIds []int32, force bool, replicationFactor int32) (*ChangeResponse, error) {
+func sendScaleRequest(port int, brokerIds []int32, partitionCount int32, force bool, replicationFactor int32) (*ChangeResponse, error) {
```

Comment:
❓ What happens when partition is 0?

Reply:
It's ignored, we can't scale down partitions anyway.